app/middlewares/userValidator.js

```diff
@@ -0,0 +1,11 @@
+const { User } = require('../models/index');
+const errors = require('../errors');
+
+exports.validate = (req, res, next) => {
```

Comment:
I don't know why do you need this middleware

Reply:
Forgot to remove it